engine/graphic/src/Texture.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Texture.h"
+Texture::Texture(char* FileName)
+{
+	m_fileName = FileName;
+	imageId = 0;
```

Comment:
I don't see any magic numbers, ImageId is being set here:
13	ilGenImages(1, &imageId);